packages/xo-server/src/xapi/index.js

```diff
@@ -2015,7 +2019,7 @@ export default class Xapi extends XapiBase {
   }
 
   async deleteVdi(vdiId) {
-    await this._deleteVdi(this.getObject(vdiId))
+    await this._deleteVdi(this.getObject(vdiId).$ref)
```

Comment:
`vdiId` is like `record.$ref` ?

Reply:
No, it's an id, which means either a `uuid` or a `ref`.